mirror_node_gateway.go

```diff
@@ -0,0 +1,99 @@
+package hedera
+
+/*-
+ *
+ * Hedera Go SDK
+ *
+ * Copyright (C) 2020 - 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+)
+
+// Function to obtain the token relationships of the specified account
+func TokenReleationshipMirrorNodeQuery(networkUrl string, id string) (map[string]interface{}, error) {
+	tokenRelationshipUrl := buildUrlParams(networkUrl, "accounts", id, "tokens")
+	return makeGetRequest(tokenRelationshipUrl)
+}
+
+// Function to obtain account info for given account ID. Return the pure JSON response as mapping
+func AccountInfoMirrorNodeQuery(networkUrl string, accountId string) (map[string]interface{}, error) {
+	accountInfoUrl := buildUrlParams(networkUrl, "accounts", accountId)
+	return makeGetRequest(accountInfoUrl)
+}
+
+// Function to obtain balance of tokens for given contract ID. Return the pure JSON response as mapping
+func ContractInfoMirrorNodeQuery(networkUrl string, contractId string) (map[string]interface{}, error) { // nolint
```

Comment:
This is not used

Reply:
Yeah, we had it defined at some point when we saw the specification differently. But I don't think we should remove it because if the mirror node query gateway where to be a full implementation it should match the api defined here:
https://mainnet-public.mirrornode.hedera.com/api/v1/docs/#/